app/renderer/ui/components/main/index.tsx

```diff
@@ -41,13 +46,36 @@ export class MainPage extends React.Component<RouteComponentProps<any>> {
         <Sidebar {...sidebarProps} pathName={this.props.location.pathname} />
         <div className={styles.main}>
           <Switch>
-            <Route path="/main/wallet" component={WalletSection.component} />
-            <Route path="/main/smartcontracts" component={SmartContractsSection.component} />
-            <Route path="/main/attestations" component={AttestationsSection.component} />
-            <Route path="/main/blockexplorer" component={BlockExplorerSection.component} />
-            <Route path="/main/community" component={CommunitySection.component} />
-            <Route path="/main/community" component={CommunitySection.component} />
-            <Route path="/" component={WalletSection.component} />
+            <PropsRoute
+              path="/main/wallet"
+              ownProps={this.props}
+              component={WalletSection.component}
+            />
+            <Route
+              path="/main/smartcontracts"
+              ownProps={this.props}
```

Comment:
If passing ownProps is required for these components, then PropsRoute should be used instead of Route